examples/generate_scenarios.py

```diff
@@ -116,5 +138,60 @@ def auto_generate_scenarios(
     Migrator().run()
 
 
+@app.command()
+def clean_env_wo_combos() -> None:
+    """
+    Function to clean up env-agent combos in the database
+    """
+    env_agent_combos = list(EnvAgentComboStorage.all_pks())
+    envs_id_in_combos = set(
+        [
+            EnvAgentComboStorage.get(env_agent_combo).env_id
+            for env_agent_combo in env_agent_combos
+        ]
+    )
+    envs = list(EnvironmentProfile.all_pks())
+    for env in envs:
+        if env not in envs_id_in_combos:
+            EnvironmentProfile.delete(env)
+
+
+@app.command()
+def upload_env_profiles(
+    filepath: str = "./data/all_environment_profile.json",
+) -> None:
+    """
+    Function to upload environment profiles from csv file
+    """
+    env_profile_list = []
+    existing_envs = pd.read_csv(
+        "./data/env_profiles_v1.csv"
+    )  # TODO: find a better way to deal with this
+    current_envs = json.load(open(filepath, "r"))
+    for key in current_envs:
+        env_profile = current_envs[key]
+        if env_profile and check_existing_envs(env_profile, existing_envs):
+            del env_profile["pk"]
+            env_profile_list.append(env_profile)
+    # randomly sample 210 envs
+    env_profile_list = random.sample(env_profile_list, 240)
```

Comment:
Just curious, why are we using random sampling?

Reply:
yeah, we only need 200ish new envs for the training purpose, I'll clean this!